walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -14,10 +14,86 @@
 
 package com.google.sps;
 
-import java.util.Collection;
+import java.util.*;
+import java.util.stream.Collectors;
 
 public final class FindMeetingQuery {
+
+  // Checks if the participants of the request are also attending the given event.
+  private static boolean sharesAttendees(Event event, MeetingRequest request) {
+    Set<String> s = event.getAttendees();
+    Collection<String> c = request.getAttendees();
+    for (String person : c) {
+      if (s.contains(person)) return true;
+    }
+    return false;
+  }
+
+  /**
+   * Returns a list of times that are compatible with a given list of events
```

Comment:
Nit: Summary fragment should be punctuated.

> the fragment is capitalized and punctuated as if it were a complete sentence.

go/java-style#s7.2-summary-fragment